_includes/templates/summaries/linkedin.html

```diff
@@ -0,0 +1,16 @@
+<h2 class="h4">
+    <span class="cf-icon cf-icon-linkedin-square"></span>
+    Follow us on LinkedIn
```

Comment:
Title case typically does not capitalize the first letter of "on", "in", "the", "and", "of", "or", or "at"